### DISCUSSION TITLE
建议默认可以以空格作为英文句子的分隔符

### DISCUSSION THREAD

**Discussion by mzhua** [YiiGuxing/TranslationPlugin#5751] [Ideas]
目前不能设置空格为第一个分隔符

**Comment by YiiGuxing** [comment]
是支持空格分隔符的啊，而且默认包含空格的

![image](https://github.com/YiiGuxing/TranslationPlugin/assets/10737066/73816d3b-ab61-441e-938e-625912eaa20e)
